setup.py

```diff
@@ -31,8 +31,13 @@
         "Programming Language :: Python :: 3.8",
         "Topic :: Utilities",
     ],
-    packages=["lnurl"],
+    packages=find_packages(exclude=["tests"]),
```

Comment:
Would be nice to avoid installing `cli` if it wasn't requested, but I don't know how to do it.

Reply:
The proper way of doing this is using "extras_require", and then install the cli using `lnurl[cli]`.

```
$ pip install lnurl[cli]
```

I can take care of this if you want.

Reply:
`pip install lnurl[cli]` should work as it is now, with `extras_require`. Or what is what you are trying to avoid installing? `pip install lnurl` won't install cli requirements.

Reply:
It won't install the requirements but will install the `cli` module anyway.